site/static/admin/config.yml

```diff
@@ -95,3 +95,11 @@ collections: # A list of collections the CMS should be able to edit
           preview: false
         fields:
           - {label: Site Title for Browser Tab, name: title, widget: string}
+          - label: Params
+            name: params
+            widget: object
+            fields:
+              - label: Open Graph (Facebook) Image
```

Comment:
Please remove Facebook here as it is only one service among many that use the OG tags.